specification/cognitiveservices/data-plane/Language/preview/2022-05-01-preview/analyzeconversations.json

```diff
@@ -0,0 +1,2383 @@
+        }
+      },
+      "required": [
+        "results"
+      ],
+      "x-ms-discriminator-value": "ConversationalPIIResults"
+    },
+    "PreBuiltResult": {
+      "properties": {
+        "errors": {
+          "type": "array",
+          "description": "Errors by conversation id.",
+          "items": {
+            "$ref": "#/definitions/ConversationError"
+          }
+        },
+        "modelVersion": {
+          "type": "string",
+          "description": "This field indicates which model is used for scoring."
+        }
+      },
+      "required": [
+        "errors",
+        "modelVersion"
+      ]
+    },
+    "ConversationPIIResults": {
+      "type": "object",
+      "description": "The result from PII detection and redaction operation for each conversation.",
+      "required": [
+        "conversations"
+      ],
+      "properties": {
+        "conversations": {
+          "type": "array",
+          "items": {
+            "allOf": [
+              {
+                "$ref": "#/definitions/ConversationPIIResult"
+              },
+              {
+                "$ref": "#/definitions/ConversationResultBase"
+              }
+            ]
+          }
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/PreBuiltResult"
+        }
+      ]
+    },
+    "ConversationPIIResult": {
+      "description": "The result from PII detection and redaction operation for each conversation.",
+      "required": [
+        "conversationItems"
+      ],
+      "properties": {
+        "conversationItems": {
+          "description": "Enumeration of PII detection and redaction operation results for all the conversation items in a conversation.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ConversationPIIItemResult"
+          }
+        }
+      }
+    },
+    "ConversationPIITaskParameters": {
+      "type": "object",
+      "description": "Supported parameters for a Conversational PII detection and redaction task.",
+      "properties": {
+        "piiCategories": {
+          "$ref": "#/definitions/ConversationPIICategories"
+        },
+        "includeAudioRedaction": {
+          "type": "boolean",
+          "description": "Flag to indicate if audio redaction is requested. By default audio redaction will not be performed.",
+          "default": false
+        },
+        "redactionSource": {
+          "type": "string",
+          "description": "For transcript conversations, this parameter provides information regarding which content type (ITN, Text, Lexical, Masked ITN) should be used for redaction. The entities detected the service in will correspond to the text type selected here.",
+          "$ref": "#/definitions/TranscriptContentType"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "common.json#/definitions/TaskParameters"
+        },
+        {
+          "$ref": "common.json#/definitions/PreBuiltTaskParameters"
```

Comment:
I think in a previous version we inherit `TaskParameters` in both `PreBuiltTaskParameters` and `CustomTaskParameters`.  Might want to double check the inheritance in the common.json file to simplify.

Reply:
yes. I removed from one and missed this one.